### DISCUSSION TITLE
Manually Trigger X-Intersect

### DISCUSSION THREAD

**Discussion by simonferndriger** [alpinejs/alpine#3825] [1. Help] [answered]
I use `<div class="loading" x-intersect:leave="visible.loadMore = false" x-intersect="visible.loadMore = true; api.reload()"></div>` to manage infinite scroll - which works fine. However, when this element becomes visible without scrolling (i.e. changing the layout with css from 2 columns to 1 column), the `x-intersect` does NOT get triggered.

How can I do that manually?

**Comment by simonferndriger** [comment]
**Closed:** On a closer look it seems to actually do work - no need for a manual trigger, sorry

**Reply by ekwoka** [reply, marked as answer]
Yup, intersection observers check the element position on initial observe, so if it meets the conditions, it's triggered right away.
